packages/cosmic-swingset/lib/ag-solo/main.js

```diff
@@ -66,7 +70,7 @@ start
     assert(basedir !== undefined, 'you must provide a BASEDIR');
     initBasedir(basedir, webport, webhost, subdir, egresses.split(','));
     await resetState(basedir);
-    // console.error(
+    // log.error(
```

Comment:
uncomment or drop

Reply:
It's code that used to exist, and we might need it again.  So, I'll leave it commented-out and add a `TODO`.